website/src/contribute/release-guide.md

```diff
@@ -1025,7 +1010,8 @@ Merge the website pull request to [list the release]({{ site.baseurl }}/get-star
 
 ### Mark the version as released in JIRA
 
-In JIRA, inside [version management](https://issues.apache.org/jira/plugins/servlet/project-config/BEAM/versions), hover over the current release and a settings menu will appear. Click `Release`, and select today’s date.
+Navigate to the [list of releases in Jira](https://issues.apache.org/jira/projects/BEAM/releases) and select the one you
+are finalizing. In the upper left, click the "Release" button.
```

Comment:
This UI doesn't exist for me, so I assume this is another place you need to be a JIRA admin?

Reply:
Does the prior UI exist?